source/Uniform/UniformDatabase.cs

```diff
@@ -62,7 +61,7 @@ public void EnterInMemoryMode()
             _metadata.Databases = newDatabases;
         }
 
-        public void LeaveInMemoryMode(bool flush = false)
+        public void LeaveInMemoryMode(bool flush = false, bool dropBeforeFlush = true)

```

Comment:
There is no backward compatibility issue with optional parameters starting from C# 4 (.NET 4.0) -- [proof link](http://stackoverflow.com/questions/3241072/is-the-optional-parameter-in-c-sharp-4-backwards-compatible). As Uniform targets .NET 4.5 ([proof link](https://github.com/paralect/uniform/blob/master/source/Uniform/Uniform.csproj#L13)), we seems to be fine with using it. Let me know if I missed something